build/source/engine/read_force.f90

```diff
@@ -296,19 +297,13 @@ subroutine read_force(istep,iGRU,iHRU_local,iHRU_global,iFile,iRead,ncid,time_da
   ncStart = (/iHRU_global,iRead/)
 
   ! get hruId
-  err = nf90_inq_varid(ncid,'hruId',varId);                   if(err/=0)then; message=trim(message)//'trouble finding HRUid variable'; return; endif
-  err = nf90_get_var(ncid,varId,hruId,start=(/iHRU_global/)); if(err/=0)then; message=trim(message)//'trouble reading HRUid variable'; return; endif
-
-  ! check that the HRUid is what we expect
-  ! NOTE: we enforce that the HRU order in the forcing files is the same as in the zLocalAttributes files (too slow otherwise)
-  if(gru_struc(iGRU)%hruInfo(iHRU_local)%hru_id /= hruId)then
-   write(message,'(a,i0,i0,a,i0,a)') trim(message)//'hruId for global HRU: ', iHRU_global, hruId, 'differs from the expected:',     &
-                                                     gru_struc(iGRU)%hruInfo(iHRU_local)%hru_id, 'in file', trim(infile)
-   write(message,'(a)') trim(message)//'order of hruId in forcing file needs to match order in zLocalAttributes.nc'
-   err=40; return
-  endif
+  err = nf90_inq_varid(ncid,'hruId',varId);                   if(err/=0)then; message=trim(message)//'trouble finding hruId variable'; return; endif
+  err = nf90_get_var(ncid,varId,hruId,start=(/iHRU_global/)); if(err/=0)then; message=trim(message)//'trouble reading hruId variable'; return; endif
 
   ! read data into forcing structure
+  ! assign the time var, convert days since reference to seconds since reference
+  forc_data(get_ixforce('time')) = varTime*secprday
+  ! other forcing var
```

Comment:
Here I moved reading `time` outside this code block because `time` only has `time` dimension but other forcing variables have `time` and `hru` dimensions.